doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,139 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
+########################
+
+This tutorial describes how to run `VxWorks`_ as the User OS on the ACRN hypervisor. We are using Kaby Lake-based
+NUC (model NUC7i5DNHE) in this tutorial. Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
+Introduction to VxWorks
+***********************
+
+VxWorks®, the industry’s leading real-time operating system, delivers all the performance, determinism, reliability,
+safety, and security capabilities you need to meet the highest standards for running the computing systems of the most
+important critical infrastructure.
+
+Steps for Using VxWorks as User OS
+**********************************
+
+#. Build VxWorks
```

Comment:
Where should you do this building work?  Later in this doc, you mention doing all this work on the NUC (in a terminal window on the Service OS), so you should mention this here as the recommended method.  (You can also mention you can do this work on some other Linux system, but extra steps will be needed to get the bootable image we're creating onto the NUC.)

Reply:
Actually we should build VxWorks on some other linux system.
I will delete the confusing words.